templates/BUILD.tpl

```diff
@@ -210,17 +210,18 @@ cc_toolchain_config(
     abi_version = "local",
     abi_libc_version = "local",
     cxx_builtin_include_directories = [
+        # Keep this list in sync with "templates/include_directory_paths_gcc"
         "/usr/include",
```

Comment:
Add `# keep sorted` for buildifier

Reply:
These will be part of "templated" starlark file, so i don't think buildifier gona work.
But I added that comment anyway as a reminder for manual sorting.